### PR TITLE
Azure Core change log for March release

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
+## 1.4.0 (2022-03-03)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Making the 1.4.0-beta.1 features GA version.
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:7]:
```suggestion
- Stabilized the beta features and changes.
```

File: sdk/core/azure-core/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Stabilized the beta features and changes.
 
 ## 1.4.0-beta.1 (2022-02-04)
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:9]: This section has an empty `### Other Changes`. Remove?

**Review by ahsonkhan — APPROVED** [review]
LGTM

File: sdk/core/azure-core/CHANGELOG.md (hunk 2 above)
Comment (vhvb1989) [reply sdk/core/azure-core/CHANGELOG.md:9]: not relevant for this PR

**Review by LarryOsterman — APPROVED** [review]
lgtm. 

:shipit:

**Comment by vhvb1989** [comment]
pipeline green but stock

**Pull request merged**